docs/howto/custom-rootfs-multiple-targets.md

```diff
@@ -0,0 +1,314 @@
+# How-to deploy a custom rootfs across multiple Windows machines with Landscape API
+
+This guide shows how to use the Landscape API to deploy a custom rootfs
+with some cloud-config user data for further initialisation of the distro
+instances via cloud-init across multiple Windows machines already registered
+with Landscape via Ubuntu Pro for WSL.
+It can be followed either on Windows (using PowerShell) or Linux (using bash).
+
+## Prerequisites
+
+- A running self-hosted Landscape server version `24.10~beta.5` or later;
+- Multiple Windows machines already registered with Landscape via Ubuntu Pro for WSL;
+- Make sure you have installed `curl` and `jq`, if you're following this guide on a Linux machine.
+- Familiarity with Bash and/or PowerShell.
```

Comment:
- Replace semi-colons with periods or else remove all of the semi-colons and periods
- You could convert "already registered with Landscape..." to a link that points to a relevant internal/external guide
- I would replace "on a Linux machine" with "using Bash"